registry/storage/blobwriter_resumable.go

```diff
@@ -0,0 +1,9 @@
+// +build !noresumabledigest
```

Comment:
why are we making these compile-time flags? is there anything in CanonicalResumableDigester that causes it from not compiling in certain cases?